frontend/src/metabase/static-viz/components/XYChart/types.ts

```diff
@@ -69,13 +67,12 @@ export type ChartSettings = {
     value: number;
     label: string;
   };
-  show_values?: boolean;
   labels: {
     left?: string;
     bottom?: string;
     right?: string;
   };
-  series_settings?: Record<string, { color?: string; title?: string }>;
+  visualization_settings: VisualizationSettings;
```

Comment:
`show_values` and `series_settings` are removed in favor of `visualization_settings` to simplify BE code.